fe/src/main/java/org/apache/doris/transaction/TransactionState.java

```diff
@@ -187,7 +187,7 @@ public String toString() {
     // this state need not to be serialized
     private Map<Long, PublishVersionTask> publishVersionTasks;
     private boolean hasSendTask;
-    private long publishVersionTime;
+    private long publishVersionTime = -1;
```

Comment:
If publishVersionTime will always be updated after finishTransaction failed, is it means that this publish job may keep runnning in doris forever?

Reply:
Yes, publish task MUST success.